python/setup.py

```diff
@@ -214,7 +236,19 @@ def config_cython():
                         os.path.join(CURRENT_DIR, "dgl", "dgl_sparse"),
                     )
                     fo.write("include dgl/dgl_sparse/%s\n" % sparse_name)
-
+                for bolt_path in glob.glob(
+                    os.path.join(dir_, "graphbolt", BOLT_LIB_PATTERN)
+                ):
+                    bolt_name = os.path.basename(bolt_path)
+                    os.makedirs(
+                        os.path.join(CURRENT_DIR, "dgl", "graphbolt"),
+                        exist_ok=True,
+                    )
+                    shutil.copy(
+                        os.path.join(dir_, "graphbolt", bolt_name),
+                        os.path.join(CURRENT_DIR, "dgl", "graphbolt"),
+                    )
+                    fo.write("include dgl/graphbolt/%s\n" % bolt_name)
```

Comment:
Can we make L226-L238 a function to avoid copy-paste?

Reply:
sure, this file relate has been moved to https://github.com/dmlc/dgl/pull/5707